templates/zerver/app/navbar.html

```diff
@@ -104,6 +104,12 @@
                                 </a>
                             </li>
                             {% endif %}
+                            <li role="presentation">
+                                <a href="https://zulip.readthedocs.io/en/latest/overview/changelog.html"  target="_blank" rel="noopener noreff" role="menuitem" class="menuitem-version">
+                                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="49.99 49.99 673.14 673.14"><path d="M688.52 150.67c0 33.91-15.23 64.04-38.44 82.31L424.79 434.17c-4.18 3.59-9.62-2.19-6.61-7.03l82.64-165.46c2.31-4.63-.69-10.33-5.44-10.33H174.86c-49.64 0-90.26-45.31-90.26-100.68 0-55.37 40.62-100.68 90.26-100.68h423.39c49.65 0 90.27 45.31 90.27 100.68zM174.86 723.13h423.39c49.64 0 90.26-45.31 90.26-100.68 0-55.37-40.62-100.68-90.26-100.68H277.73c-4.75 0-7.76-5.7-5.44-10.33l82.64-165.46c3.01-4.83-2.42-10.62-6.61-7.03L123.04 540.14c-23.21 18.27-38.44 48.4-38.44 82.31 0 55.37 40.62 100.68 90.26 100.68z" /></svg>
+                                    {{ _('Zulip ' + (zulip_version[:-4] if zulip_version.endswith('+git') else zulip_version) ) }}
```

Comment:
See the comments I've posted in the original PR -- I want a function in say `zerver/lib/home.py` that generates a URL as well as a string to use that will be passed into this template -- we'll end up having like 5 conditionals in that function, I think.

Reply:
Yes that's why I created the variable `zulip_version_name`. I need to know just this. Working on this, will update soon.

Reply:
Hello, I have created the function in `context_processors.py`, I have currently made a simple logic for the name to be shown. I don't know the conditionals, either you can add them accordingly or tell about them.
Also what URL are you talking about to be generated in the function? I didn't get that.